docs/content/getting-started/_index.md

```diff
@@ -82,10 +82,26 @@ To use the Drasi Getting Started Dev Container, you will need to install:
 - Visual Studio Code [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
 - [docker](https://www.docker.com/get-started/)
 
+#### Recommended Docker Resources
+
+For optimal performance with the Drasi Dev Container, we recommend configuring Docker with the following minimum resources:
+
+- **CPU**: 3 cores or more
+- **Memory**: 4 GB or more
+- **Swap**: 1 GB or more
+- **Disk**: 272 GB available space 
```

Comment:
272 GB of disk space seems excessive? Mine was working with 50GB.

Reply:
I will try to run the tutorials again with 50 GB